### ISSUE TITLE
Usable- Bot Framework Web Chat- Card Input]: In High Contrast dark themes, show date picker, show time picker controls are not visible.

### ISSUE THREAD

**Issue by manojs5** [microsoft/BotFramework-WebChat#5342]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:
Color sensitive users who use high contrast themes will be affected as they cannot identify current focused element, cannot effectively navigate and access the controls if the focus indicator is not visible clearly. 

### Note: 
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated, Suggested action

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label 
3. and then select open web chat in new window link at bottom.
4. Navigate to white label experience chat page and type Card Input word in type your message text box.
5. Navigate to the search results and fill details and then navigate to submit results and show card results.
6. Verify whether the controls are visible in high contrast dark themes or not.

### What do you expect?

In High Contrast dark themes, show date picker, show time picker controls should be visible.

### What actually happened?

 In High Contrast dark themes, show date picker, show time picker controls are not visible.

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/ef696150-d90d-42be-abb4-67dd5d03000f)
![Image](https://github.com/user-attachments/assets/801b2212-048d-4773-aeec-c88f1d71d131)
![Image](https://github.com/user-attachments/assets/954ece70-4ed1-4215-bf35-b6aea1573387)
![Image](https://github.com/user-attachments/assets/d02e89bb-aa69-4309-8bb6-ebe6b1c36646)

**Comment by manojs5** [comment]
#A11ySev4;#A11yUsable;#Accessibility;#Benchmark;#BotFrameworkWebChat-Oct24;#HCL;#Win11-Edge(Chromium);

**Comment by Kirank915** [comment]
@OEvgeny This issue is marked as external to Adaptive cards, could you please confirm if your team will fix the issue, or we need to share this issue to adaptive cards team?

**Comment by Kirank915** [comment]
@OEvgeny This issue is marked as external to Adaptive cards, could you please confirm if your team will fix the issue, or we need to share this issue to adaptive cards team?

**Comment by Kirank915** [comment]
@OEvgeny could you please confirm if your team will fix the issue, or we need to share this issue to adaptive cards team?

**Comment by Kirank915** [comment]
@OEvgeny could you please confirm if your team will fix the issue, or do we need to share this issue with the adaptive cards team?

**Comment by OEvgeny** [comment]
So, this is adaptive cards issue:

Repro steps:
1. Goto https://adaptivecards.io/designer/ with high contrast theme turned on
2. Paste the following content into the center bottom field:
```json
{
  "type": "AdaptiveCard",
  "version": "1.0",
  "speak": "<s>Adaptive Card inputs example</s>",
  "body": [
    {
      "type": "TextBlock",
      "size": "medium",
      "weight": "bolder",
      "text": "Input.Text elements",
      "horizontalAlignment": "center"
    },
    {
      "type": "Input.Text",
      "placeholder": "Name",
      "style": "text",
      "maxLength": 0,
      "id": "SimpleVal",
      "speak": "What is your name?"
    },
    {
      "type": "Input.Text",
      "placeholder": "Homepage",
      "style": "url",
      "maxLength": 0,
      "id": "UrlVal",
      "speak": "What is your homepage?"
    },
    {
      "type": "Input.Text",
      "placeholder": "Email",
      "style": "email",
      "maxLength": 0,
      "id": "EmailVal",
      "speak": "What is your email address?"
    },
    {
      "type": "Input.Text",
      "placeholder": "Phone",
      "style": "tel",
      "maxLength": 0,
      "id": "TelVal",
      "speak": "What is your phone number?"
    },
    {
      "type": "Input.Text",
      "placeholder": "Comments",
      "style": "text",
      "isMultiline": true,
      "maxLength": 0,
      "id": "MultiLineVal",
      "speak": "What comments do you have?"
    },
    {
      "type": "Input.Number",
      "placeholder": "Quantity",
      "min": -5,
      "max": 5,
      "value": "",
      "id": "NumVal",
      "speak": "How many do you want?"
    },
    {
      "type": "Input.Date",
      "placeholder": "Due Date",
      "id": "DateVal",
      "speak": "What is the due date?"
    },
    {
      "type": "Input.Time",
      "placeholder": "Start time",
      "id": "TimeVal",
      "speak": "What time to start?"
    },
    {
      "type": "TextBlock",
      "size": "medium",
      "weight": "bolder",
      "text": "Input.ChoiceSet",
      "horizontalAlignment": "center"
    },
    {
      "type": "Input.ChoiceSet",
      "label": "What color do you want?",
      "style": "compact",
      "choices": [
        {
          "title": "Red",
          "value": "1",
          "isSelected": true
        },
        {
          "title": "Green",
          "value": "2"
        },
        {
          "title": "Blue",
          "value": "3"
        }
      ],
      "id": "SingleSelectVal"
    },
    {
      "type": "Input.ChoiceSet",
      "label": "What color do you want?",
      "choices": [
        {
          "title": "Red",
          "value": "1",
          "isSelected": true
        },
        {
          "title": "Green",
          "value": "2"
        },
        {
          "title": "Blue",
          "value": "3"
        }
      ],
      "id": "CompactSelectVal"
    },
    {
      "type": "Input.ChoiceSet",
      "label": "What colors do you want?",
      "isMultiSelect": true,
      "choices": [
        {
          "title": "Red",
          "value": "1",
          "isSelected": true
        },
        {
          "title": "Green",
          "value": "2"
        },
        {
          "title": "Blue",
          "value": "3",
          "isSelected": true
        }
      ],
      "id": "MultiSelectVal"
    },
    {
      "type": "TextBlock",
      "size": "medium",
      "weight": "bolder",
      "text": "Input.Toggle",
      "horizontalAlignment": "center"
    },
    {
      "type": "Input.Toggle",
      "title": "isCool (True/False)",
      "valueOn": "true",
      "valueOff": "false",
      "id": "IsCool",
      "speak": "Is it cool?"
    },
    {
      "type": "Input.Toggle",
      "title": "IsMale (valueOn/valueOff)",
      "valueOn": "Male",
      "valueOff": "Female",
      "id": "gender",
      "speak": "Are you male or female?"
    }
  ],
  "actions": [
    {
      "type": "Action.OpenUrl",
      "url": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
      "title": "Open Url"
    },
    {
      "type": "Action.Submit",
      "data": {
        "id": "1234567890"
      },
      "title": "Submit"
    },
    {
      "type": "Action.Http",
      "method": "POST",
      "url": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
      "body": "This is the body with Gender={{gender.value}}",
      "title": "POST"
    },
    {
      "type": "Action.ShowCard",
      "card": {
        "type": "AdaptiveCard",
        "body": [
          {
            "type": "TextBlock",
            "text": "Seattle vs Panthers"
          },
          {
            "type": "ColumnSet",
            "columns": [
              {
                "type": "Column",
                "size": "auto",
                "items": [
                  {
                    "type": "Image",
                    "altText": "No image available",
                    "size": "small",
                    "url": "https://www.bing.com/th?id=AR_3934aff63dd6ff941034090a4b9d12e1&w=50&h=50&dpr=2&pid=AppEx",
                    "horizontalAlignment": "center"
                  },
                  {
                    "type": "TextBlock",
                    "weight": "bolder",
                    "text": "CAR",
                    "horizontalAlignment": "center"
                  }
                ]
              },
              {
                "type": "Column",
                "size": "stretch",
                "items": [
                  {
                    "type": "TextBlock",
                    "text": "Dec 4",
                    "horizontalAlignment": "center"
                  },
                  {
                    "type": "TextBlock",
                    "text": "Final",
                    "horizontalAlignment": "center"
                  },
                  {
                    "type": "TextBlock",
                    "size": "large",
                    "text": "7 - 40",
                    "horizontalAlignment": "center"
                  }
                ]
              },
              {
                "type": "Column",
                "size": "auto",
                "items": [
                  {
                    "type": "Image",
                    "altText": "No image available",
                    "size": "small",
                    "url": "https://www.bing.com/th?id=AR_95e44d654968e8de2336993a1cf8a3cf&w=50&h=50&dpr=2&pid=AppEx",
                    "horizontalAlignment": "center"
                  },
                  {
                    "type": "TextBlock",
                    "weight": "bolder",
                    "text": "SEA",
                    "horizontalAlignment": "center"
                  }
                ]
              }
            ]
          },
          {
            "type": "Input.Text",
            "placeholder": "enter comment",
            "style": "text",
            "maxLength": 0,
            "id": "CommentVal"
          }
        ],
        "actions": [
          {
            "type": "Action.Submit",
            "title": "OK"
          }
        ],
        "speak": "The Seattle Seahawks beat the Carolina Panters 40-7"
      },
      "title": "Show Card"
    }
  ]
}
```

Observe if icons align with the theme applied:

![Image](https://github.com/user-attachments/assets/e49af357-9165-4940-b836-6cc8c08346b9)
